perceval/runtime/rpc_handler.py

```diff
@@ -50,7 +50,7 @@ class RPCHandler:
 
     """
 
-    def __init__(self, name, url, token):
```

Comment:
There's another implementation of `RPCHandler` in `perceval.providers.scaleway`. Could you make the same changes there, please (if it's easy)?